console/src/app/modules/form-field/field/form-field.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="cnsl-form-field-wrapper" (click)="_control.onContainerClick && _control.onContainerClick($event)">
+  <ng-content select="cnsl-label"></ng-content>
+  <div class="cnsl-rel" #inputContainer>
+    <ng-content></ng-content>
+    <ng-content select="cnslSuffix"></ng-content>
+  </div>
+  <ng-container *ngIf="help$ && help$ | async as help">
+    <div [ngSwitch]="help.type" class="cnsl-form-field-subscript-wrapper">
+      <div *ngSwitchCase="'hints'" class="cnsl-form-field-hint-wrapper" [@transitionMessages]="_subscriptAnimationState">
+        <ng-content select="cnsl-hint"></ng-content>
+      </div>
+      <div class="cnsl-form-field-error-wrapper" [@transitionMessages]="_subscriptAnimationState"></div>
```

Comment:
either delete this line or add `<ng-content select="cnsl-error"></ng-content>` here, or add the cnsl-form-field-error-wrapper class to the ngSwitchCase below

Reply:
done